src/utils.js

```diff
@@ -23,4 +23,26 @@ export function elementWidth(el) {
  */
 export function lockBodyScrolls(lock) {
   document.body.style.overflow = lock ? 'hidden' : '';
-}
\ No newline at end of file
+}
+
+/**
+ * takes a number and returns it as a human readable number
+ * exp (10000 = 10k)
+ * @param {num} num is the number
+ * @returns returns the the number as human readable number
+ */
+export function readableNumber(num) {
+  if (num > 1000000) {
+    return `${(num / 1000000).toFixed(1)}m`;
+  } else if (num > 100000) {
+    return `${Math.floor(num / 1000)}k`;
+  } else if (num >= 10000) {
+    return `${(num / 1000).toFixed(1)}k`;
+  } else if (num < 10000) {
+    return digitGrouping(num);
```

Comment:
I would add a comment here explaining why only this case needs grouping.

Reply:
Done